centos_ci.sh

```diff
@@ -292,14 +292,40 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+function perform_docs_publish() {
+  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
+  DOC_USER=$(echo $REPO | cut -d"/" -f4)
```

Comment:
DOC_USER seems to be not quite what this variable is about. More like repo owner, right?

Reply:
Ya you are right. Will update it.